applications/nrf_desktop/src/modules/Kconfig.power_manager

```diff
@@ -1,44 +0,0 @@
-#
-# Copyright (c) 2018 Nordic Semiconductor
-#
-# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
-#
-
-menu "Power manager"
-
-config DESKTOP_POWER_MANAGER_ENABLE
-	bool "Enable power management"
-	select PM_DEVICE
-	help
-	  Enable power management, which will put the device to low-power mode
-	  if it is idle.
-
-if DESKTOP_POWER_MANAGER_ENABLE
```

Comment:
This setting has been there for 3 years: https://github.com/nrfconnect/sdk-nrf/commit/4dbfda77cf20c509fe8fbab298803bb1cd7b1de4
i.e. since release 0.1.0.

Is it really a good idea to just remove this setting without any deprecation first ?

@Rallare and @OeySa feel free to give opinions on this.

Reply:
We had cases where options were migrated to CAF. We could deprecate I think - essentially leave this option and select one in caf. I am however unsure if this will really be huge problem to fix during update.
What I mean there is no code that depends on it - there is no associated API. So the only thing to do is to look at diff and accept the change.

Reply:
No, but we could have getting started guides, blog posts, etc. that refers to the existing application and settings.

All of a sudden, all such additional information is void, and will cause confusion for the reader.

Reply:
also, tech support might be use to asking, have you enabled `XYZ` if something is not working for the user, and suddenly this setting is no longer there.

We should really improve how we rename things, and see it from other people / users perspective.

Reply:
Then it would be much better to have just a file with:

    config DESKTOP_POWER_MANAGER_ENABLE
    bool "Enable power management"
    depends NCS_DEPRECATED
    help
      This option is deprecated. Use CAF_POWER_MANAGER instead.

Reply:
Renaming and changing kconfig entries will, in general, force a active change for developers that are updating their own project; thus forcing them to actively have to port their own project. This is something we want to avoid in general, or atleast kept at a backwards compatible level.

In this case is for a specific application, so as long as this change is well documented/handled I do not see any problem. I think something like you mention is perfectly fine when added to the applications own kconfig file, @rakons